### PR TITLE
fix columnar batch

### PR DESCRIPTION
**Pull request by birdstorm**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

File: core/src/main/scala/com/pingcap/tispark/write/TiBatchWrite.scala [hunk 1]
```diff
@@ -405,7 +405,10 @@ class TiBatchWrite(
     logger.info(s"sampleSize=$sampleSize")
 
     val sampleData =
-      rdd.map(_._1).sample(withReplacement = false, sampleSize.toDouble / count).collect()
+      rdd
```
[the remark this answers is not in the document]
Comment (birdstorm) [reply core/src/main/scala/com/pingcap/tispark/write/TiBatchWrite.scala:408]: due to scala compiler check.

File: tikv-client/src/main/java/com/pingcap/tikv/types/ArrayType.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
+/*
+ *
+ * Copyright 2017 PingCAP, Inc.
```
Comment (marsishandsome) [inline tikv-client/src/main/java/com/pingcap/tikv/types/ArrayType.java:3]:
```suggestion
 * Copyright 2020 PingCAP, Inc.
```

**Review by marsishandsome — APPROVED** [review]
LGTM

**Pull request merged**